keps/sig-ui/5008-headlamp/README.md

```diff
@@ -235,11 +240,17 @@ The "Design Details" section below is for the real
 nitty-gritty.
 -->
 
-The proposal is to ship the Headlamp experience as a new Kubernetes Desktop
-and Kubernetes in-cluster UI.
+The proposal is to continue the development of Headlamp as part of the
+Kubernetes project, under the auspices of Kubernetes SIG UI.
```

Comment:
Can you link to https://github.com/headlamp-k8s/headlamp/issues/2897 for visibility?

Reply:
I have updated the PR as sugested.